ui/src/components/SearchResult/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Framer from 'framer-motion';
+import * as OutlineIcons from '@heroicons/react/24/outline';
+import React from 'react';
+
+import * as Components from '@/components';
+
+type Props = {
+    children?: React.ReactNode;
+    className?: string;
+    linkDestination: string;
+};
+
+const SearchResult: React.FC<Props> = (props): React.ReactElement => {
+    return (
+        <Framer.motion.div
+            initial={{ opacity: 0 }}
+            animate={{ opacity: 1 }}
+            exit={{ opacity: 0 }}
+            transition={{ type: 'tween', duration: 0.35 }}
+        >
+            <Components.Link
+                href={props.linkDestination}
+                className={`
```

Comment:
:joy: tailwind vs `<Components.Link href={props.linkDestination} className="search-link">`

Reply:
:') something to consider.